build-aux/xlib.sh

```diff
@@ -0,0 +1,51 @@
+# common code base for fa_build/fa{gcc,db,llvm} and sl_build/sl{gcc,gdb,llvm}
```

Comment:
I would not rename the file as ```xlib.sh``` as one might think it refers to https://en.wikipedia.org/wiki/Xlib -- what about ```cclib.sh``` instead?

Of course, no need to resubmit...